tox.ini

```diff
@@ -7,6 +7,10 @@ skip_missing_interpreters = True
 [testenv]
 deps =
     setuptools-scm  # to get cached setuptools-scm on "sdist-make" step
+    pytest
+    pytest-asyncio
+    pytest-cov
+    pytest-recording
```

Comment:
These deps should be listed in `testing` extras in `setup.cfg`.

Reply:
If you will use tox from console like _tox -p all_ this modules needs in config.

Reply:
`tox` installs everything that listed in `extras`. So it's better to keep all testing modules in `setup.cfg`.

Reply:
Strangely did not work for me but I have now double-checked and it seems like yes tox really uses extras.testing
I will remove this commit

Reply:
Looks like `venv` created by tox was cached. `tox -p all --recreate` might help.